core/chains/evm/config/toml/defaults/Linea_Mainnet.toml

```diff
@@ -0,0 +1,21 @@
+ChainID = '59144'
+# Block time 12s, finality < 60m
+FinalityDepth = 300
+# Blocks are only emitted when a transaction happens / no empty blocks
+NoNewHeadsThreshold = '0'
+
+[GasEstimator]
+BumpPercent = 40
+PriceMin = '400 mwei'
+
+[Transactions]
+# increase resend time to align with finality
+ResendAfterThreshold = '3m'
+
+# set greater than finality depth
+[HeadTracker]
+HistoryDepth = 350
+
+# Linea uses 200 blocks to estimate gas price
```

Comment:
200 seems very large.
Is this depth recommended by Linea?

Reply:
Indeed. I see ~15 txs per block on Mainnet, which translates to 3000 txs for the estimator. In comparison, ETH Mainnet has ~150txs per block and the fallback value is 8.

Reply:
At analysis stage the Linea docs mentioned that gas was calculated on the last 200 blocks. For this reason `BlockHistorySize` was set to 200. Should be noted that this detail has now been removed from the docs.